src/ranged.cpp

```diff
@@ -378,6 +378,10 @@ int player::fire_gun( const tripoint &target, int shots, item& gun )
         }
     }
 
+    if( shots <= 0 ) {
+        debugmsg( "Attempted to fire zero or negative shots using %s", gun.tname().c_str() );
```

Comment:
This debugmsg still triggers.
* Spawn a vehicle
* Add flamethrower
* Add between 0 and 100 gasoline (ie. more than 0 but less than 100)
* Get debugmsg every turn the flamethrower tries to fire